### PR TITLE
Adding ioncube loader to the container

### PR DESCRIPTION
**Pull request by emberdyn**
This is a great tool.  Wondering if we can add ioncube loader to the container?  

Thanks.

### CODE REVIEW

**Comment by emberdyn** [comment]
@markshust  - Is it a possibility to add this in so that we can use ioncube for local development?

File: images/php/7.2/Dockerfile [hunk 1]
```diff
@@ -44,6 +44,20 @@ RUN docker-php-ext-install \
   xsl \
   zip
 
+RUN cd /tmp \
```
Comment (markshust) [inline images/php/7.2/Dockerfile]: Can you keep just this single `RUN` command and remove all of the other updates? We don't want to add backports to the repo to add libsodium, it's already added in libsodium-dev above.

File: images/php/7.1/Dockerfile [hunk 2]
```diff
@@ -44,6 +44,22 @@ RUN docker-php-ext-install \
   xsl \
   zip
 
+
```
Comment (markshust) [inline images/php/7.1/Dockerfile]: Let's remove this extra blank line, keep just the `RUN` here and remove all of the other updates

File: images/php/7.2/Dockerfile (hunk 1 above)
Comment (emberdyn) [reply images/php/7.2/Dockerfile]: Whoops.  My bad.  Thanks for catching that.  I have made the requested changes.  Thanks!

File: images/php/7.1/Dockerfile (hunk 2 above)
Comment (emberdyn) [reply images/php/7.1/Dockerfile]: I have made the requested changes.  Thank you!

**Comment by markshust** [comment]
Great, all merged! I'll rebuild a new set of images shortly.

**Comment by avra911** [comment]
@markshust , @emberdyn , please see my PR: https://github.com/markshust/docker-magento/pull/219.

Thank you!

**Comment by markshust** [comment]
Great, all merged. I'll see what I can do about getting another build out shortly.

**Pull request merged**